README.md

```diff
@@ -29,17 +30,16 @@ Please make sure you run `sandbox-ce` commands in the same directory where you p
 #### Install
 
 ```
-sandbox-ce install /some/path
+sandbox-ce install /path/to/repositories
 ```
 
-This will create admin user and initialize superset.
-
-After the initialization, the components superset, gitbase, bblfsh and other dependencies will be started.
+This will initialize the source{d} Sandbox to analyze the given Git repositories.
```

Comment:
do we need to mention that it's Sandbox?

Reply:
Changed in f001a5d